### PR TITLE
Fixes #349: Newly created entities were no selectable from the viewport

### LINKED ISSUE #349: Newly created entities with geometry are not selectable in the viewport

### PR DESCRIPTION
**Pull request by fernandojsg**
- Allow newly created entities to be selected on the viewport aswell as its helpers
- Start using `componentinitialized` for newly created components.
- Created `empty.html` to use for testing empty scenes

### CODE REVIEW

File: example/empty.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Example Scene</title>
+    <script src="js/aframe.min.js"></script>
+  </head>
+  <body>
+    <a-scene stats>
+    </a-scene>
+    <script src="http://localhost:3333/build/aframe-inspector.js"></script>
```
Comment (dmarcos) [inline example/empty.html:11]: Is this to have an empty scene to edit?

File: src/components/components/Sidebar.js [hunk 2]
```diff
@@ -35,13 +35,20 @@ export default class Sidebar extends React.Component {
     Events.emit('selectedEntityComponentChanged', event.detail);
   }
 
+  componentCreated = (event) => {
+    Events.emit('selectedEntityComponentCreated', event.detail);
```
Comment (dmarcos) [inline src/components/components/Sidebar.js:39]: Should we remove camel case from event names for consistency?

File: example/empty.html (hunk 1 above)
Comment (fernandojsg) [reply example/empty.html:11]: Yes, many times I want to try the inspector with an empty scene and it's annoying to edit the current example or delete every element.

File: src/components/components/Sidebar.js (hunk 2 above)
Comment (fernandojsg) [reply src/components/components/Sidebar.js:39]: Yep, I've opened an issue for that https://github.com/aframevr/aframe-inspector/issues/357

Comment (fernandojsg) [reply src/components/components/Sidebar.js:39]: And here you have it: https://github.com/aframevr/aframe-inspector/pull/358

**Comment by ngokevin** [comment]
r+

**Pull request merged**